rollup-http/rollup-init/rollup-init

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-busybox sh -c "/opt/cartesi/bin/rollup-http-server $*"
+busybox sh -c "/usr/bin/rollup-http-server $*"
```

Comment:
If we don't namespace our tools anymore, do we still need full paths?

Reply:
Maybe we can change this in the future, but for now I think explicit is better.